integrations/micrometer/cdi/src/test/java/io/helidon/integrations/micrometer/cdi/HelloWorldTest.java

```diff
@@ -41,6 +41,7 @@
  */
 @HelidonTest
 @AddBean(HelloWorldResource.class)
```

Comment:
CatchAllExceptionMapper was removed from Helidon, and test requires it.

Reply:
I'm wondering whether we can move this class to a common location and either include it explicitly or create a subclass of it with the `@Provider` annotation for discovery (thus sharing the base class code). Mostly to avoid repeating this code, and partly in case someone wants to port an app from 3.x to 4.x that uses it.

Reply:
I kept it in the module it was: helidon/microprofile/server
But I had to make public. In the javadoc I wrote that:
`It is not configured by default in Helidon but it is provided for explicit usage.`